app/src/main/java/com/sopt/kakaobank/presentation/home/HomeFragment.kt

```diff
@@ -1,11 +1,78 @@
 package com.sopt.kakaobank.presentation.home
 
+import android.graphics.Rect
+import android.os.Bundle
+import android.telephony.PhoneNumberUtils.formatNumber
+import android.util.TypedValue
+import android.view.View
+import android.widget.AdapterView
+import android.widget.AdapterView.OnItemClickListener
+import androidx.databinding.DataBindingUtil.setContentView
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.sopt.kakaobank.R
 import com.sopt.kakaobank.core.base.BindingFragment
 import com.sopt.kakaobank.databinding.FragmentHomeBinding
+import com.sopt.kakaobank.databinding.ItemBankBook1Binding
+import com.sopt.kakaobank.presentation.history.HistoryFragment
+import java.text.NumberFormat
+import java.util.Locale
 
 class HomeFragment : BindingFragment<FragmentHomeBinding>(R.layout.fragment_home) {
+
+    private val viewModel: HomeViewModel by viewModels()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.homeViewModel = viewModel
+        binding.lifecycleOwner = viewLifecycleOwner
+        initRecyclerView()
+    }
+
+    private fun initRecyclerView() {
```

Comment:
```suggestion
    private fun initHomeItemAdapter() {
```
init[NameView]Adapter : 리사이클러뷰 어댑터 설정